backend/municipal-services/ingestion-service/app/api/endpoints/template_generation.py

```
@@ -0,0 +1,112 @@
+logger = AppLogger().get_logger()
+
+from dotenv import load_dotenv
+
+load_dotenv()
+mdms_url = os.getenv("MDMS_URL")
+
+@router.get('/facilityIngestion',
+            summary='Generate facility ingestion template Excel file with schema and boundary codes',
+            response_description="Returns Excel template with facility schema and boundary codes")
+async def get_facility_ingestion_template(
+        facility_service: FacilityTemplateService = Depends(),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)